examples/cookies/server.go

```diff
@@ -0,0 +1,63 @@
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/erikdubbelboer/fasthttp"
+	"github.com/themester/fcookiejar"
+)
+
+func main() {
+	fmt.Println(
+		fasthttp.ListenAndServe(":8080", cookieHandler),
+	)
+}
+
+func cookieHandler(ctx *fasthttp.RequestCtx) {
+	cookie := fasthttp.AcquireCookie()
+	cookie1 := fasthttp.AcquireCookie()
+	cookie2 := fasthttp.AcquireCookie()
+	// do not forget to release
+	// IMPORTANT: If you are going to use cookiejar DO NOT release manually.
+	// CookieJar package releases cookies automatically.
+	//
+	//defer fasthttp.ReleaseCookie(cookie)
+	//defer fasthttp.ReleaseCookie(cookie1)
+	//defer fasthttp.ReleaseCookie(cookie2)
+
+	cookie.SetDomain("make.fasthttp.great.again")
+	cookie.SetKey("key")
+	cookie.SetExpire(time.Now().Add(time.Hour))
+	cookie.SetPath("/")
+	cookie.SetValue("value")
+
+	cookie1.SetDomain("make.fasthttp.great.again")
+	cookie1.SetKey("use")
+	cookie1.SetPath("/path")
+	cookie1.SetValue("fasthttp")
+
+	cookie2.SetKey("hello")
+	cookie2.SetValue("world")
+
+	// You can use SetCookie calls
+	/*
+		ctx.Response.Header.SetCookie(cookie)
+		ctx.Response.Header.SetCookie(cookie1)
+		ctx.Response.Header.SetCookie(cookie2)
+	*/
```

Comment:
I would just use this code for the example. Using the cookiejar here just adds extra allocations and extra lines of code that add nothing. Calling `ctx.Response.Header.SetCookie` for each cookie is much simpler than creating a cookiejar and calling `cookies.Put` for each cookie.

Reply:
Do you suggest to skip cookiejar example for server?

Reply:
I'm wondering if cookiejar is really that useful at all. What can you really do with it that you can't easily do with the functions already provided in fasthttp?

Reply:
I made it keeping in mind the client usage. You can see the usage of cookiejar in https://github.com/LibreLABUA/duac/blob/master/http.go#L26
Using fasthttp functions I'd have to parse cookie array or pointer to cookie array.

Reply:
Ah I get it. That shows much better use of the cookiejar where you want future requests to include cookies from previous responses. An example like that is much better, I suggest you rewrite this example to something like that. What you are doing in this pull request doesn't require a cookiejar at all.

But in this case I would also suggest improving the cookiejar code. A proper cookiejar should seperate cookies per domain, your cookiejar now combines cookies from all domains. Also `Set-Cookie` with an expire time in the past should remove the cookie from the jar. For a good example see the buildin cookiejar: https://golang.org/pkg/net/http/cookiejar/